client/app/views/contacts/components/datapoints.coffee

```diff
@@ -1,13 +1,19 @@
+t = require 'lib/i18n'
+
 ContactDatapointView = require 'views/contacts/components/datapoint'
 
 CONFIG = require('config').contact
 
 
-module.exports = class ContactDatapointsView extends Mn.CollectionView
+module.exports = class ContactDatapointsView extends Mn.CompositeView
 
     tagName: ->
         if @options.cardViewModel.get 'edit' then 'fieldset' else 'ul'
 
+    template: (data) ->
+        name = t "datapoints section #{data.name}"
+        "<legend>#{name}</legend>" if data.edit
```

Comment:
I had to compile to JS to understand what this function return.
Please make it explicit for both case (data.edit truthy/falsy)  :-)